CompetionTaskMars/Utilities/JsonReader.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace CompetionTaskMars.Utilities
+{
+    public static class JsonReader
+    {
+        public static Credentials GetCredentials()
+        {
+            var jsonFilePath = "C:\\repo\\CompetionTaskMars\\CompetionTaskMars\\CompetionTaskMars\\TestData\\testData.json"; 
+            var jsonData = File.ReadAllText(jsonFilePath);
+            return JsonConvert.DeserializeObject<Credentials>(jsonData);
+        }
+    }
+
+    public class Credentials
+    {
+        public string Email { get; set; }
```

Comment:
would create new class for objects as separate files, maybe create a package called model or pojo and then create Credentials class and any other class that you need there